### DISCUSSION TITLE
[Screenshot] Add customizability for the filename format

### DISCUSSION THREAD

**Discussion by farcast3r** [runelite/runelite#17572] [Ideas]
A dropdown with some options (flipping date format for UK/EU) or a text field to customize the filename output 

Concept:
![Untitled-1](https://github.com/runelite/runelite/assets/156135703/c2584f9e-6851-4cca-8e7e-7eb50705eb86)

**Comment by i-afk-crab** [comment]
Well, I went searching for a way to do this, and eventually found my way to this post. I really hope this is added one day, because `month/day/year` is not the format used by anything else stored on my PC. As you suggested, I'd like it if I could change it to `day/month/year`.
